### PR TITLE
Technicave patch 1

### PR DESCRIPTION
**Pull request by technicave**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6571"></a>

### CODE REVIEW

File: data/technicave.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Aryan Tiwari",
+  "type": "personal",
+  "bio": "| Full Stack Developer | Chief Creative Officer at Showwcase | Creator at Technicave | Loves to Contributing to the Community |",
+  "avatar": "https://github.com/technicave.png",
```
Comment (khushalarora) [inline data/technicave.json:5]: You can remove this line, it is not required anymore :+1: 
```suggestion
```

File: data/technicave.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "Name": "Checkout Github",
+      "url": "https://github.com/technicave",
+      "icon": "FaGithub"
+    },
+    {
+      "Name": "Follow on LinkedIn",
+      "url": "https://www.linkedin.com/in/technicave/",
+      "icon": "FaLinkedin"
+    },
```
Comment (khushalarora) [inline data/technicave.json:21]: Please remove the extra **' Comma '** from this line :+1: 
```suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @technicave, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**